### PR TITLE
Add sourcejar create task

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

**Comment by loosebazooka** [comment]
@patflynn

source : https://discuss.gradle.org/t/how-to-configure-gradle-to-build-a-source-jar/3209

File: build.gradle [hunk 1]
```diff
@@ -22,6 +22,12 @@ subprojects {
     tasks.withType(JavaCompile) { options.encoding = 'UTF-8' }
     group = 'com.google.gct'
 
+    task jarSources(type:Jar) {
```
Comment (patflynn) [inline build.gradle]: isn't src jar generation available by default?

Comment (loosebazooka) [inline build.gradle]: I can't really find any other way to do this. But by default no sources jar is generated.

Comment (patflynn) [inline build.gradle]: LGTM

**Pull request merged**